app/src/main/java/com/example/lapnguyen/weather_3/screen/mainscreen/MainPresenter.java

```diff
@@ -0,0 +1,60 @@
+package com.example.lapnguyen.weather_3.screen.mainscreen;
+
+import android.app.Activity;
+import android.content.Context;
+import android.icu.text.SimpleDateFormat;
+import android.location.Location;
```

Comment:
ERROR: Unused import: android.location.Location. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports